pages/api/serverClient.ts

```diff
@@ -9,6 +9,14 @@ export const postLogin = async (path?: string, body?: any) => {
     return data;
 }
 export const getBlogs = async (path?: string) => {
```

Comment:
getBlogs, getPopular and getCategory are they same function we need to de duplicate them